tools-v2/pkg/cli/command/curvebs/status/client/client.go

```diff
@@ -0,0 +1,236 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-04-27
+ * Author: Xinlong-Chen
+ */
+
+package client
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/client"
+	config "github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/exp/slices"
+)
+
+const (
+	clientExample = `$ curve bs status client`
+)
+
+type ClientCommand struct {
+	basecmd.FinalCurveCmd
+	metrics []*basecmd.Metric
+	rows    []map[string]string
+}
+
+const (
+	PROCESS_CMD_SUBURI = "/vars/process_cmdline"
+	VERSION_SUBURI     = "/vars/curve_version"
+)
+
+const (
+	PROCESS_CMD_KEY = "process"
+	VERSION_KEY     = "version"
+	COUNT_KEY       = "count"
+)
+
+const (
+	kProcessNebdServer string = "nebd-server"
+	kProcessQemu       string = "qemu"
+	kProcessPython     string = "python"
+	kProcessOther      string = "other"
+)
+
+var _ basecmd.FinalCurveCmdFunc = (*ClientCommand)(nil) // check interface
+
+func NewClientCommand() *cobra.Command {
+	return NewStatusClientCommand().Cmd
+}
+
+func (cCmd *ClientCommand) AddFlags() {
+	config.AddHttpTimeoutFlag(cCmd.Cmd)
+}
```

Comment:
```suggestion
func (cCmd *ClientCommand) AddFlags() {
    config.AddBsMdsFlagOption(pCmd.Cmd)
	config.AddRpcRetryTimesFlag(pCmd.Cmd)
	config.AddRpcTimeoutFlag(pCmd.Cmd)
	config.AddHttpTimeoutFlag(cCmd.Cmd)
}
```

Reply:
It calls the subcommand list client, and these parameters need to be passed into the subcommand.